src/core/xref.js

```diff
@@ -11,6 +11,11 @@ import {
   showInlineWarning,
 } from "./utils";
 import { openDB } from "idb";
+import { pub } from "./pubsubhub";
+
+const profiles = {
+  w3c: ["HTML", "INFRA", "URL", "WEBIDL", "DOM", "FETCH"],
```

Comment:
I decided "w3c" is probably not the right name... "web-platform" is more accurate.

Reply:
OK, I will make the changes. 

Although "web-platform" is a bit vague to me (almost every W3C and WHATWG profile could be included in web platform - so what would other profiles names be in future?).

Reply:
No stress, spec editors have a pretty good idea what constitutes “the platform”. 

There could be: “performance” for performance specs. And “a11y” for Accessibility specs... “css” for css specs and so on.